sle-stack-config.yaml

```diff
@@ -0,0 +1,9 @@
+public-buildpacks-ci-robots-branch: sle
+docker-stack-repository: staging.registry.howdoi.website:443/splatform/rootfs-sles
```

Comment:
I know this is outside the scope of this PR, but the image name should be `splatform/rootfs-sle12` instead of `...-sles`.

Reply:
Good point. Maybe @jimmykarily can follow up on this? This is part of the `sle-rootfs-publish` pipeline I think.

Reply:
Yes, it is configurable through `docker-internal-rootfs`. I think the reason was that we are only going to have one supported sle stack at any given time so that would be the latest supported version (whatever that may be). I will add a task to discuss this on Monday.

Reply:
> we are only going to have one supported sle stack at any given time

That sounds weird to me. At least during development of the next version we will have both the current version and the upcoming one simultaneously. And even when we release the new version, do we expect all customers to switch to the new version on release day because the old version will be unsupported on that very day?

Reply:
This is the internal registry. The customers won't ever have to access this, it is only used by the pipeline to build the product. The docker images for all previous versions will still exist in the public registry so the customers should be able to build any previous release with no problem. Regarding development, I agree that we might need to build CAP for 2 versions. Maybe we should use the TAG of the image to define the version (which right now is not used since we always push as `latest`).

Reply:
Yes, we should use tags to describe the specific build of the rootfs, which is more than just the OS version.

I still don't understand the reluctance to use different namespaces for different versions of the OS.  This is all to help us to avoid mistakes. It will also help us with automation because we won't have to parse tags to find e.g. the latest sle15 rootfs automatically.

Reply:
Then convention for docker images is to use the tag for version (See ubuntu:14.04, opensuse:42.2 etc). I'm not reluctant, I just want to know why we want to break the convention otherwise someone else might come with a different idea in 1 month and rename it back. It's reasoning that I seek for, sorry for the confusion.

Reply:
I can't guarantee that somebody else won't come and make us change things again...

The reasoning from my part is that the tag should be the version of the rootfs itself (which is different from, and will change more often, than the version of the OS it is based on). See e.g. the tags for `cflinuxfs2`: https://hub.docker.com/r/cloudfoundry/cflinuxfs2/tags/ These are not Ubuntu:Trusty version numbers, but tags from the `cflinuxfs2` repo: https://github.com/cloudfoundry/cflinuxfs2/releases

So I could see us use the same version/tag of the rootfs and have corresponding images for both sle12 and sle15. Of course these versions could be combined into a single tag, but that becomes messy quickly.

I'm sure upstream will have separate `cflinuxfs3` images at some point soon, so we would simply follow their practice.

In general having predictable naming schemes is a big advantage, there are already too many combination of things. If you have stacks called `sle12` and `sle15`, why would you want to intermingle their rootfs images into a single namespace, and then have to parse and interpret tags to tell them apart again?

Reply:
That's good reasoning, thanks :) Renaming on the way.